generators/app/index.js

```diff
@@ -255,6 +259,12 @@ module.exports = JhipsterGenerator.extend({
                 this.generatorType = 'server';
                 this.skipUserManagement = configOptions.skipUserManagement = true;
             }
+            if (this.applicationType == 'uaa') {
+                this.skipClient = true;
+                this.generatorType = 'server';
+                this.skipUserManagement = configOptions.skipUserManagement = false;
+                //this.authenticationType = configOptions.authenticationType = 'uaa';
```

Comment:
yes the commented line, if its not required plz remove it. But from your code flow I assume its ok to set `authenticationType` type to uaa when `applicationType` is uaa